File: workspace-servidor/flotix2021/src/main/java/com/flotix/response/bean/ServerResponseAlerta.java
package com.flotix.response.bean;

import java.util.List;

import com.flotix.dto.AlertaDTO;

public class ServerResponseAlerta {

	private String idAlerta = null;

	private List<AlertaDTO> listaAlerta = null;

	private ErrorBean error = new ErrorBean();

	public ServerResponseAlerta() {
		super();
	}

	public String getIdAlerta() {
		return idAlerta;
	}

	public void setIdAlerta(String idAlerta) {
		this.idAlerta = idAlerta;
	}

	public List<AlertaDTO> getListaAlerta() {
		return listaAlerta;
	}

	public void setListaAlerta(List<AlertaDTO> listaAlerta) {
		this.listaAlerta = listaAlerta;
	}

	public ErrorBean getError() {
		return error;
	}

	public void setError(ErrorBean error) {
		this.error = error;
	}
}

File: workspace_presentacion/Flotix2021/Flotix2021/ModelResponse/ServerResponseAlerta.cs
using Flotix2021.ModelDTO;
using System.Collections.Generic;

namespace Flotix2021.ModelResponse
{
    class ServerResponseAlerta
    {
        private string _idAlerta;

        public string idAlerta
        {
            get { return _idAlerta; }
            set { _idAlerta = value; }
        }

        private List<AlertaDTO> _listaAlerta;

        public List<AlertaDTO> listaAlerta
        {
            get { return _listaAlerta; }
            set { _listaAlerta = value; }
        }

        private ErrorBean _error;

        public ErrorBean error
        {
            get { return _error; }
            set { _error = value; }
        }
    }
}
